odoo/models.py

```diff
@@ -251,6 +251,93 @@ def expand_ids(id0, ids):
 }
 
 
+# THE DEFINITION AND REGISTRY CLASSES
```

Comment:
Commit _[IMP] core: document registry classes and field sharing_  should be retitled if you want to keep this in a separate commit (looks like a comment move only)